mlf_core/create/templates/common_mlflow_files/cookiecutter.json

```diff
@@ -0,0 +1,17 @@
+{
+  "common_mlflow": "common_mlflow_files",
+  "full_name": "mlf_coreusername",
+  "email": "mlf_core@dummy.com",
+  "project_name": "projectname",
+  "project_slug": "mlf_core_template",
+  "project_slug_no_hyphen": "mlf_core_template",
+  "version": "0.1.0",
+  "license": "MIT",
+  "project_short_description" : "A mlf_core template",
+  "language": "Language",
+  "domain": "domain",
+  "github_username": "HomerGithub",
+  "creator_github_username": "BartGithub",
+  "framework": "mlf_core_framework"
```

Comment:
I decided to add a `framework` since this was the only difference in the README of all mlflow templates, so I was able to refactor them out